src/pages/SupportPage.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import FooterWrapper from 'src/components/FooterWrapper';
+import { PageBox, PageBoxVariants } from 'src/components/PageBox';
+import TopBar from 'src/components/TopBar';
+import * as Nav from 'src/libs/nav';
+import * as Style from 'src/libs/style';
+import { SupportResourceList } from 'src/support/SupportResourceList';
+
+interface SupportPageProps {
+  queryParams: {
+    resourceType: string | undefined;
+    resourceId: string | undefined;
+  };
+}
+
+const SupportPage = (props: SupportPageProps) => {
+  const selectedType = props.queryParams.resourceType;
+  const resourceId = props.queryParams.resourceId;
+  const breadcrumbs = `Support > ${selectedType}`;
+
+  return (
+    <FooterWrapper>
+      <TopBar title='Support' href={Nav.getLink('support')}>
+        {resourceId && (
+          <div style={Style.breadcrumb.breadcrumb}>
+            <div style={Style.noWrapEllipsis}>{breadcrumbs}</div>
+            <div style={Style.breadcrumb.textUnderBreadcrumb}>{resourceId}</div>
+          </div>
+        )}
+      </TopBar>
+      <PageBox variant={PageBoxVariants.light}>
```

Comment:
I was wrong-- this one DOES need the main role. The problem was the other one because it created 2 in the same page. I'll see if I can figure out how to suggest the change:

<PageBox role='main' variant={PageBoxVariants.light}>